Application/EdFi.Ods.Api/Authorization/PoliciesAuthorizationHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+#if NETCOREAPP
+using Microsoft.AspNetCore.Authorization;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EdFi.Ods.Api.Authorization
+{
+    public class PoliciesAuthorizationHandler : AuthorizationHandler<CustomUserRequireClaim>
+    {
+        protected override Task HandleRequirementAsync(
+            AuthorizationHandlerContext context,
+            CustomUserRequireClaim requirement)
+        {
+            if (context.User == null || !context.User.Identity.IsAuthenticated)
+            {
+                context.Fail();
+                return Task.CompletedTask;
+            }
+
+            var hasClaim = context.User.Claims.Any(x => x.Type == requirement.ClaimType);
```

Comment:
you may want to look at EdFiAuthorizationFilter and see  what we need to do to validate the claim is valid

Reply:
EdFiAuthorizationFilter has lots of dependencies that are in netstandard 2.1 so it would require converting of those dependencies to .net core app need to clarify.